pinot-hadoop/src/main/java/com/linkedin/pinot/hadoop/job/TableConfigConstants.java

```diff
@@ -0,0 +1,5 @@
+package com.linkedin.pinot.hadoop.job;
+
+public class TableConfigConstants {
```

Comment:
this class is not needed

Reply:
Reasoning is to separate out constants that we get from table config vs hadoop constants; there will be more coming. It looks sparse for now because it's piecemeal/haven't put in the majority of table config constants. I can combine them if you prefer though!